packages/wallets/polkadot.js/src/polkadot.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  Chain,
+  type ConnectWalletParams,
+  WalletOption,
+  setRequestClientConfig,
+} from "@swapkit/helpers";
+import { getWalletForChain } from "./helpers";
+
+const POLKADOT_SUPPORTED_CHAINS = [Chain.Polkadot, Chain.Chainflip] as const;
```

Comment:
As we are not supporting Chainflip in this wallet, you should probably remove it from this array

Reply:
done